webui/react/src/hooks/useModal/Experiment/useModalExperimentMove.tsx

```diff
@@ -341,19 +329,16 @@ const useModalExperimentMove = ({ onClose }: Props): ModalHooks => {
       if (!destSettings.workspaceId)
         updateDestSettings({ projectId: undefined, workspaceId: sourceWorkspaceId });
       setSourceProjectId(sourceProjectId);
-      if (!workspaces.length) fetchWorkspaces();
```

Comment:
This line isn't necessary since the workspaces are always fetched when this modal is mounted.